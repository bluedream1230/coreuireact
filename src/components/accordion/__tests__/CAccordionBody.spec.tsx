import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CAccordionBody } from '../../../index'

test('loads and displays CAccordionBody component', async () => {
  const { container } = render(<CAccordionBody>Test</CAccordionBody>)
  expect(container).toMatchSnapshot()
})

test('CAccordionBody customize', async () => {
  const { container } = render(<CAccordionBody className="bazinga">Test</CAccordionBody>)
  expect(container.firstChild).toHaveClass('bazinga')
  expect(container.firstChild).toHaveClass('accordion-body')
  expect(container).toMatchSnapshot()
})
