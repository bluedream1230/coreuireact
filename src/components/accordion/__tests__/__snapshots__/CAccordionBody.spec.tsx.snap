// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CAccordionBody customize 1`] = `
<div>
  <div
    class="accordion-body bazinga"
  >
    Test
  </div>
</div>
`;

exports[`loads and displays CAccordionBody component 1`] = `
<div>
  <div
    class="accordion-body"
  >
    Test
  </div>
</div>
`;
